yarn-project/stdlib/src/database-version/version_manager.ts

```diff
@@ -59,6 +59,10 @@ export class DatabaseVersion {
       .transform(({ schemaVersion, rollupAddress }) => new DatabaseVersion(schemaVersion, rollupAddress));
   }
 
+  public toString(): string {
+    return this.schemaVersion.toString();
```

Comment:
minor fix for a printing of [object Object]

Reply:
Worth overriding `[inspect.custom]` as well